CHANGELOG.md

```diff
@@ -10,6 +11,7 @@
 * **validate** will no longer fail on playbooks calling subplaybooks that have a higher `fromVersion` value, if  calling the subplaybook has `skipifunavailable=True`.
 * Fixed an issue where relative paths were not accessed correctly.
 * Running any `demisto-sdk` command in a folder with a `.env` file will load it, temporarily overriding existing environment variables.
+* Fixed an issue where **find_type** function didn't recognize exported incident types.
```

Comment:
the bold part is usually the SDK command users use, I suppose it's mostly format that uses `find_type`?

Reply:
I'm not sure, also validate, upload, secrets, generic, lint and other